### DISCUSSION TITLE
How to rotate the scene under a certain point?

### DISCUSSION THREAD

**Discussion by chenguanga** [pelicanmapping/osgearth#1917] [Support Q&A] [answered]
The EarthManipulator's rotate center is its focalpoint，but I want  to roate the scene under a certain point(the mouse position or any model center), Are there solutions?

**Comment by gwaldron** [comment, marked as answer]
You can use the `Viewpoint` object to set a focal point around which the camera will rotate. We commonly call this "tethering." Activate the viewpoint be calling `EarthManipulator::setViewpoint`. Hope this helps.
